pkg/build/api/validation/validation.go

```diff
@@ -500,11 +500,14 @@ func validateTrigger(trigger *buildapi.BuildTriggerPolicy, buildFrom *kapi.Objec
 	return allErrs
 }
 
-func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path) field.ErrorList {
+func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path, allowEnv bool) field.ErrorList {
```

Comment:
(and if you want to refactor that code so that all the trigger output gets printed from within describeBuildTriggers instead of the way it's done currently, i would not complain.)